.github/workflows/aws.yml

```
@@ -28,14 +28,14 @@
-    branches: ["main","dev"]
+    branches: [ "main","dev" ]
   workflow_dispatch:
 
 
 env:
   AWS_REGION: ap-northeast-2                   # set this to your preferred AWS region, e.g. us-west-1
   ECR_REPOSITORY: otboo           # set this to your Amazon ECR repository name
-  ECS_SERVICE: otboo                # set this to your Amazon ECS service name
+  ECS_SERVICE: otboo1                # set this to your Amazon ECS service name
   ECS_CLUSTER: obtoo-ecs-cluster
   ECS_TASK_DEFINITION: .aws/obtoo-def-revision2.json # set this to the path to your Amazon ECS task definition
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)